File: ClientSide/src/app/pages/students/models/studentModel.ts
export class StudentModel {
    
    public id: number;
    public code: string;
    public name: string;
    public surname: string;

    public studyProgrammeId: number;
    public language: number;
    public learningForm: number;
    public status: number;

    public phone1: string;
    public phone2: string;
    public email1: string;
    public email2: string;
    public skype: string;
    public comment: string;


    constructor(
        id: number,
        code: string,
        name: string,
        surname: string,

        studyProgrammeId: number,
        language: number,
        learningForm: number,
        status: number,

        phone1: string,
        phone2: string,
        email1: string,
        email2: string,
        skype: string,
        comment: string
    ) {
        this.id = id;
        this.code = code;
        this.name = name;
        this.surname = surname;

        this.studyProgrammeId = studyProgrammeId;
        this.language = language;
        this.learningForm = learningForm;
        this.status = status;

        this.phone1 = phone1;
        this.phone2 = phone2;
        this.email1 = email1;
        this.email2 = email2;
        this.skype = skype;
        this.comment = comment;
    }
}

File: ServerSide/SAMS.BusinessLogic/Models/Students/StudentModel.cs
using SAMS.BusinessLogic.Entities.Enumerations;
using System;
using System.Collections.Generic;
using System.Text;

namespace SAMS.BusinessLogic.Models.Students
{
    public class StudentModel
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }

        public int StudyProgrammeId { get; set; }

        public int Language { get; set; }
        public int LearningForm { get; set; }
        public int Status { get; set; }

        public string Phone1 { get; set; }
        public string Phone2 { get; set; }
        public string Email1 { get; set; }
        public string Email2 { get; set; }
        public string Skype { get; set; }
        public string Comment { get; set; }
    }
}
